### DISCUSSION TITLE
How to debug a mobile Tauri app?

### DISCUSSION THREAD

**Discussion by georgiai1** [tauri-apps/tauri#11699] [Q&A] [answered]
Hey just created a new mobile Tauri + Nuxt app but got a white screen with no errors in the console. Running the desktop version works flawlessly.
Running the same mobile demo but with Vue only everything works as expected. 
How to start to unravel the problem to find where the culprit is?

I am on windows + android platform

For example how can I inspect the web view in the phone is there a guide somewhere?

**Comment by FabianLars** [comment, marked as answer]
> For example how can I inspect the web view in the phone is there a guide somewhere?

https://v2.tauri.app/develop/#opening-the-web-inspector-1

**Comment by georgiai1** [comment]
Thanks this got me further with my debugging for nuxt.js

There is something wrong internally with nuxt because I get errors

11-18 01:04:14.193 15907 15907 E Tauri/Console: File:  - Line 138 - Msg: Uncaught TypeError: Cannot redefine property: postMessage
11-18 01:04:14.194 15907 15907 E Tauri/Console: File:  - Line 2 - Msg: Uncaught TypeError: Cannot redefine property: metadata
11-18 01:04:14.194 15907 15907 E Tauri/Console: File:  - Line 25 - Msg: Uncaught TypeError: Cannot redefine property: __TAURI_PATTERN__
11-18 01:04:14.194 15907 15907 E Tauri/Console: File:  - Line 5 - Msg: Uncaught TypeError: Cannot redefine property: path

**Reply by FabianLars** [reply]
hmm this could also be caused by https://github.com/tauri-apps/wry/pull/1418 but it shouldn't break any apps 🤔

**Reply by georgiai1** [reply]
Hmm maybe but my debugging skills about Tauri end here, if you guide me more I can continue investigating

` ![image](https://github.com/user-attachments/assets/474f43fd-25de-4375-a048-9466fe5b28d2)`

**Reply by georgiai1** [reply]
Not sure what I did but for the first time the app is working

Even with the errors, the tauri.localhost switcharoo is doing something funky with nuxt. Using localhost:3000 directly works

EDIT: it worked only once, couldn't repeat the working state

**Reply by georgiai1** [reply]
How to disable the tauri.localhost switch and use the localhost:3000 by default on mobile?

**Comment by metkm** [comment]
This is still an issue. It's trying to use tauri.localhost
